plugins/org.eclipse.elk.alg.packing.rectangles/src/org/eclipse/elk/alg/packing/rectangles/util/DrawingData.java

```diff
@@ -0,0 +1,212 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Kiel University and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     Kiel University - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.elk.alg.packing.rectangles.util;
+
+/**
+ * Class that offers instances to save information about a possible drawing. Can be used to return multiple values that
+ * are calculated for a possible drawing in a method for example.
+ * 
+ * @author dalu
+ */
+public class DrawingData {
+    //////////////////////////////////////////////////////////////////
+    // FIELDS
+    /** Scale measure of the given drawing. */
+    private double scaleMeasure;
+    /** Width of the drawing. */
+    private double drawingWidth;
+    /** Height of the drawing. */
+    private double drawingHeight;
+    /** Drawing area. */
+    private double area;
+    /** Aspect ratio of the drawing. */
+    private double aspectRatio;
+    /** Desired aspect ratio. */
+    private double dar;
+    /** Indicates what placement option this data belongs to or whether it belongs to a whole drawing. */
+    private DrawingDataDescriptor plaOpt;
+    /**
+     * If a this object contains information about a possible drawing, this field contains the potential x-coordinate
+     * for the new rectangle.
+     */
+    private double nextXcoordinate;
+    /**
+     * If a this object contains information about a possible drawing, this field contains the potential y-coordinate
+     * for the new rectangle.
+     */
+    private double nextYcoordinate;
+
+    //////////////////////////////////////////////////////////////////
+    // CONSTRUCTORS
+    /**
+     * Creates a new object saving parameters of a drawing.
+     * 
+     * @param dar
+     *            desired aspect ratio.
+     * @param drawingWidth
+     *            drawing width.
+     * @param drawingHeight
+     *            drawing height.
+     * @param placementOption
+     *            placement Option.
+     */
+    public DrawingData(final double dar, final double drawingWidth, final double drawingHeight,
```

Comment:
Call the other constructor (line 86) with default values for the unused members here.